tests/test_find.py

```diff
@@ -12,9 +13,35 @@ def repo_dir(request):
     return Path('tests', request.param)
 
 
-def test_find_template(repo_dir):
+@pytest.fixture()
+def env(context):
+    """Fixture return the env generated from context."""
+    return create_env_with_context(context)
+
+
+@pytest.mark.parametrize(
+    "repo_name,context,expected",
+    [
+        ("fake-repo-pre", {}, '{{cookiecutter.repo_name}}'),
+        (
+            "fake-repo-pre2",
+            {
+                'cookiecutter': {
+                    '_jinja2_env_vars': {
+                        'variable_start_string': '{%{',
+                        'variable_end_string': '}%}',
+                    }
+                }
+            },
+            '{%{cookiecutter.repo_name}%}',
+        ),
+    ],
+)
+def test_find_template(repo_name, env, expected):
```

Comment:
I've added a parametrized test here: `template with custom jinja strings but folder with default jinja strings`
where the jinja configuration is overriden with `{%{`, but the template folder is uses `{{`

In this case cookiecutter should not be able to find the template folder because it is not accurately named, so it should fail with `NonTemplatedInputDirException`